README.md

```diff
@@ -10,10 +10,21 @@ Clone this repository and install the dependencies.
 
 ## Testing
 
-Runs integration tests using arlocal. Note - these test write temporary files to your local disc.
+Unit tests and integration tests are located in the [tests] directory. Both unit and integration tests are written using [Jest].
 
-- `yarn test` - runs full test suite
-- `yarn test -t 'auctions'` - runs tests that match this spec name (e.g. `auctions.test.ts`)
+### Integration Tests
+
+Runs integration tests against deployed Smartweave contract using [Warp SDK].
+
+- `yarn test:integration` - runs full test suite
+- `yarn test:integration -t 'auctions'` - runs tests that match this spec name (e.g. `auctions.test.ts`)
```

Comment:
Does using the filename as the example confuse the fact that it actually looks at test names to decide which to run (rather than filenames? or is it both?)?

Reply:
you're right - it's test names. i'll add a reference for how to run individual files and update

Reply:
modified in [ae31e0b](https://github.com/ar-io/testnet-contract/pull/113/commits/ae31e0b914906328e1c11934976d6284a9ed7d77)